components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/KeyManagerConnectorConfiguration.java

```diff
@@ -77,6 +84,29 @@ public default String getDefaultConsumerKeyClaim() {
      * This method returns keymanager endpoint configurations.
      */
     public default List<ConfigurationDto> getEndpointConfigurations() {
-        return Collections.emptyList();
+
+        List<ConfigurationDto> configurationDtos = new ArrayList<>();
+        configurationDtos.add(new ConfigurationDto("client_registration_endpoint", "Client Registration Endpoint",
```

Comment:
do we need underscores in the variable names? can follow the camelCase approach as usual?